NASA-IMPACT/admg-backend
# "To be developed" placeholder page

## What has been built?
I added a page that gives a message to the user that that portion of the website has not yet been developed.  The main goal here was to provide some more clarity since I anticipate a broader group of people exploring the website on their own after the meeting on Friday.

## How was it done?
I added:
* A new template `to_be_developed.html`
* A function-based view definition `to_be_developed` to `views.py`
* A new path to `admin_ui/urls.py` for the path `tbd` and named it `to-be-developed`
* Hooked up the new page to the side navigation in `api_app/base.html`

## How can it be tested?
Go to http://localhost:8000/admin/drafts and click on the "Other Items" or "Inventory Activity" side navigation tabs.  These tabs should show the new placeholder page.
